cmd/ci-op-configs-mirror/main.go

```diff
@@ -144,3 +149,10 @@ func privatePromotionConfiguration(promotion *api.PromotionConfiguration) {
 func strP(str string) *string {
 	return &str
 }
+
+func isIntegrationImageStream(name string) bool {
```

Comment:
Technically you need to walk every configuration we care about and see which image streams we promote to, right? Not everyone promotes to something prefixed with `4.`

Reply:
It should be sufficient to use the same condition that is used in `promotesOfficialImages` check - we only mirror configs that promtoe into official imagestreams.

But in general yes, any reference to any IS that is a promotion target of the private org config should be changed.